docs/guide/permissions.md

```diff
@@ -42,109 +42,110 @@ The more Role assignments and complex Policies you add for a given User, the mor
-| `comparison` | `view` | view version comparison |
-| `personalization`    | `view`       | view scenario configuration and results for selected SiteAccesses |
-|| `edit`|modify scenario configuration for selected SiteAccesses|
-| `segment`</br>[[% include 'snippets/commerce_badge.md' %]] | `read`|load Segment information|
-|| `create`|create Segments|
-|| `update`|update Segments|
-|| `remove`|remove Segments|
-|| `assign_to_user` |assign Segments to Users|
-| `segment_group`</br>[[% include 'snippets/commerce_badge.md' %]] | `read` |load Segment Group information|
-|| `create` |create Segment Groups|
-|| `update` |update Segment Groups|
-|| `remove` |remove Segment Groups|
-| `product` | `create` |create a product|
-|| `view` |view products|
-|| `edit` |edit a product|
-|| `delete` |delete a product|
-| `product_type` | `create` |create a product type|
-|| `view` |view product types|
-|| `edit` |edit a product type|
-|| `delete` |delete a product type|
-| `attribute_group` | `create` |create an attribute group|
-|| `view` |view attribute groups|
-|| `edit` |edit an attribute group|
-|| `delete` |delete an attribute group|
-| `commerce` | `currency` |manage currencies|
-|| `region` |manage regions|
-| `customer_group` | `create` |create a customer group|
-|| `view` |view customer groups|
-|| `edit` |edit a customer group|
-|| `delete` |delete a customer group|
-|`siso_policy`</br>[[% include 'snippets/commerce_badge.md' %]]|`checkout`|access the checkout process|
-||`edit_invoice`|edit invoice address|
-||`edit_delivery`|edit delivery address|
-||`delegate`|access delegate screen|
-||`dashboard_view`|access the Back Office cockpit|
-||`forms_profile_edit`|access the user profile|
-||`manage_orders`|access Order Management screen|
-||`manage_prices`|work in Price management tab|
-||`manage_stock`|work in Stock management tab|
-||`manage_shipping_costs`|work in Shipping costs management tab|
-||`manage_config`|access eCommerce configuration settings|
-||`lostorder_list`|access the lost orders in the Back Office|
-||`lostorder_manage`|manage lost orders|
-||`lostorder_process`|process lost orders|
-||`orderhistory_view`|view Order history|
-||`quickorder`|access the quick order|
-||`read_basket`|see the basket|
-||`write_basket`|modify the basket (add, update, delete)|
-||`see_product_price`|see product prices in the catalog|
-|`siso_customercenter`</br>[[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]|`approve`|approve baskets in the customer center|
-||`buy`|buy as the customer center user|
-||`view`|access the customer center user management|
+| Module                                                                                                                | Function                                                | Effect                                                                                                                                                                                  |
```

Comment:
Note: reformatting the table should not be included in this PR, because it's out of scope (and table fixes should cover all afflicted branches)